### ISSUE TITLE
when custom training set is given, the model is not learning. character error rate, line accuracy, word error rate always 0.000% how can this be solved? ---tf1.8

### ISSUE THREAD

**Issue by nitindantu** [lamhoangtung/LineHTR#7]
Please have a look at the FAQ section in the README - maybe your question is already answered there. 
Only issues concerning the repositories code will be answered.
The following questions will not be answered:
* How to convert dataset X into IAM format?
* How to modify the model to recognize text-lines/more characters/...?
* General/theoretical questions regarding (handwritten) text recognition.

If you create a new issue, please provide the following information:

1. Versions
* TensorFlow version
* Python version
* Operating system

2. Issue
* Which result/error did you get?
* If you think the result is wrong - what result did you expect instead?
* How to reproduce the issue?
* Provide all necessary data

**Comment by lamhoangtung** [comment]
Sorry for the late reply, many things can go wrong here, probably with your dataset. Please provide more details
